[glib] Failed to build

```shell
[136/1212] Linking target glib/libglib-2.0.so.0.6700.0
FAILED: glib/libglib-2.0.so.0.6700.0 
cc  -o glib/libglib-2.0.so.0.6700.0 glib/libglib-2.0.so.0.6700.0.p/deprecated_gallocator.c.o glib/libglib-2.0.so.0.6700.0.p/deprecated_gcache.c.o glib/libglib-2.0.so.0.6700.0.p/deprecated_gcompletion.c.o glib/libglib-2.0.so.0.6700.0.p/gpattern.c.o gl/usr/local/lib64/gcc/x86_64-cros-linux-gnu/10.2.0/../../../../x86_64-cros-linux-gnu/bin/ld: subprojects/sysprof/src/libsysprof-capture/libsysprof-capture-4.a(sysprof-collector.c.o): in function `sysprof_collector_get.part.0':
sysprof-collector.c:(.text+0x104): undefined reference to `__stack_chk_guard'
/usr/local/lib64/gcc/x86_64-cros-linux-gnu/10.2.0/../../../../x86_64-cros-linux-gnu/bin/ld: sysprof-collector.c:(.text+0x2a3): undefined reference to `__stack_chk_guard'
/usr/local/lib64/gcc/x86_64-cros-linux-gnu/10.2.0/../../../../x86_64-cros-linux-gnu/bin/ld: subprojects/sysprof/src/libsysprof-capture/libsysprof-capture-4.a(sysprof-collector.c.o):sysprof-collector.c:(.text+0x5ad): more undefined references to `__stack_chk_guard' follow
collect2: error: ld returned 1 exit status
[141/1212] Compiling C object glib/tests/array-test.p/array-test.c.o
ninja: build stopped: subcommand failed.
glib failed to install: `ninja -C build` exited with 1[136/1212] Linking target glib/libglib-2.0.so.0.6700.0
FAILED: glib/libglib-2.0.so.0.6700.0 
cc  -o glib/libglib-2.0.so.0.6700.0 glib/libglib-2.0.so.0.6700.0.p/deprecated_gallocator.c.o glib/libglib-2.0.so.0.6700.0.p/deprecated_gcache.c.o glib/libglib-2.0.so.0.6700.0.p/deprecated_gcompletion.c.o glib/libglib-2.0.so.0.6700.0.p/gpattern.c.o gl/usr/local/lib64/gcc/x86_64-cros-linux-gnu/10.2.0/../../../../x86_64-cros-linux-gnu/bin/ld: subprojects/sysprof/src/libsysprof-capture/libsysprof-capture-4.a(sysprof-collector.c.o): in function `sysprof_collector_get.part.0':
sysprof-collector.c:(.text+0x104): undefined reference to `__stack_chk_guard'
/usr/local/lib64/gcc/x86_64-cros-linux-gnu/10.2.0/../../../../x86_64-cros-linux-gnu/bin/ld: sysprof-collector.c:(.text+0x2a3): undefined reference to `__stack_chk_guard'
/usr/local/lib64/gcc/x86_64-cros-linux-gnu/10.2.0/../../../../x86_64-cros-linux-gnu/bin/ld: subprojects/sysprof/src/libsysprof-capture/libsysprof-capture-4.a(sysprof-collector.c.o):sysprof-collector.c:(.text+0x5ad): more undefined references to `__stack_chk_guard' follow
collect2: error: ld returned 1 exit status
[141/1212] Compiling C object glib/tests/array-test.p/array-test.c.o
ninja: build stopped: subcommand failed.
glib failed to install: `ninja -C build` exited with 1
```

Here's what I get without ```-fno-stack-protector```:
```
sysprof-capture-cursor.c:(.text+0x141): undefined reference to `__stack_chk_guard'
/usr/local/lib64/gcc/x86_64-cros-linux-gnu/10.2.0/../../../../x86_64-cros-linux-gnu/bin/ld: sysprof-capture-cursor.c:(.text+0x21b): undefined reference to `__stack_chk_guard'
/usr/local/lib64/gcc/x86_64-cros-linux-gnu/10.2.0/../../../../x86_64-cros-linux-gnu/bin/ld: subprojects/sysprof/src/libsysprof-capture/libsysprof-capture-4.a(sysprof-collector.c.o): in function `sysprof_collector_free.part.0':
sysprof-collector.c:(.text+0x50): undefined reference to `__stack_chk_guard'
/usr/local/lib64/gcc/x86_64-cros-linux-gnu/10.2.0/../../../../x86_64-cros-linux-gnu/bin/ld: subprojects/sysprof/src/libsysprof-capture/libsysprof-capture-4.a(sysprof-collector.c.o): in function `sysprof_collector_get.part.0':
sysprof-collector.c:(.text+0x104): undefined reference to `__stack_chk_guard'
/usr/local/lib64/gcc/x86_64-cros-linux-gnu/10.2.0/../../../../x86_64-cros-linux-gnu/bin/ld: sysprof-collector.c:(.text+0x2a3): undefined reference to `__stack_chk_guard'
/usr/local/lib64/gcc/x86_64-cros-linux-gnu/10.2.0/../../../../x86_64-cros-linux-gnu/bin/ld: subprojects/sysprof/src/libsysprof-capture/libsysprof-capture-4.a(sysprof-collector.c.o):sysprof-collector.c:(.text+0x5ad): more undefined references to `__stack_chk_guard' follow
collect2: error: ld returned 1 exit status
[140/1215] Compiling C object glib/tests/array-test.p/array-test.c.o
ninja: build stopped: subcommand failed.
glib failed to install: `ninja -C build` exited with 1
```
Good call on trying ```-fstack-protector```

--- a/packages/glib.rb
+++ b/packages/glib.rb
@@ -15,7 +15,7 @@ class Glib < Package
   depends_on 'util_linux'
   depends_on 'six'
   
-  ENV['CFLAGS'] = "-fno-stack-protector"
+  ENV['CFLAGS'] = "-fstack-protector"
   def self.build
     system "meson #{CREW_MESON_OPTIONS} \
     -Dselinux=disabled \

